scripts/generate/templates/SUBSCRIPTION_MANAGEMENT/react.template.js

```diff
@@ -158,7 +158,7 @@ function Create() {
           <TextField
             label="Plan title"
             value={planTitle}
-            onAfterChange={setPlanTitle}
+            onChange={setPlanTitle}
```

Comment:
should these be `onInput` instead of `onChange`? It seems like `onAfterChange` => `onInput` since they are the same functionality

Reply:
It's correct that `onAfterChange` becomes `onInput` now but we use `onChange` here is on purpose. `onChange` is a common usage and doesn't affect performance because it doesn't send message through the bridge for every input.

Reply:
I thought `onAfterChange` => `onChange`, and polaris `onChange` => `onInput`?

Reply:
Discussed offline with Alan, we are in the same page. This PR is good.